functions_framework_builder/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## 0.3.1-dev
+
```

Comment:
Document the change: allow `functions_framework` versions xyz...

Reply:
Should it or should it not match the [pubspec](https://github.com/GoogleCloudPlatform/functions-framework-dart/blob/main/functions_framework_builder/pubspec.yaml#L2). You were using `-dev` before. Or are you saying we need to bump *and* publish the builder to stay in sync with the ff version?